packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/FIAObjectTranslator.m

```diff
@@ -288,4 +289,86 @@ + (SKPaymentDiscount *)getSKPaymentDiscountFromMap:(NSDictionary *)map
   return discount;
 }
 
++ (nullable SKPaymentTransactionMessage *)convertTransactionToPigeon:
+    (SKPaymentTransaction *)transaction API_AVAILABLE(ios(12.2)) {
+  if (!transaction) {
+    return nil;
+  }
+  SKPaymentTransactionMessage *msg = [SKPaymentTransactionMessage
+            makeWithPayment:[self convertPaymentToPigeon:transaction.payment]
+           transactionState:[self convertTransactionStateToPigeon:transaction.transactionState]
+        originalTransaction:transaction.originalTransaction
```

Comment:
Does this need to null check, since this method returns a nil value if it is null already?